contracts/did-stellar-registry/src/contract.rs

```
@@ -0,0 +1,394 @@
+        if !ok {
+            return false;
+        }
+    }
+    true
+}
+
+/// Accepts only URLs that begin with the literal prefix `https://` and are
+/// no longer than `MAX_URL_LEN`. Does not parse or validate the host.
+fn is_valid_https_url(s: &String) -> bool {
+    let len = s.len();
+    if len < 8 || len > MAX_URL_LEN {
```

Clippy (1.95.0)

[warning] manual `!RangeInclusive::contains` implementation

(warning)